rednotebook/gui/browser_cef.py

```diff
@@ -107,11 +101,30 @@ def _embed_browser(self):
             window_info = cef.WindowInfo()
             self._win32_handle = self.get_handle()
             window_info.SetAsChild(self._win32_handle)
-            self._browser = cef.CreateBrowserSync(window_info, url="file:///dummy/")
-            self._browser.SetClientHandler(_RequestHandler())
+            self._browser = cef.CreateBrowserSync(window_info, url=self.NOTEBOOK_URL)
+            self._browser.SetClientCallback("OnBeforeBrowse", self.on_before_browse)
+            self._browser.SetClientCallback("OnAddressChange", self.on_address_change)
             self.load_html(self._initial_html)
             self._initial_html = None
 
+        @GObject.Signal(name="on-url-clicked", arg_types=(str,))
+        def url_clicked_signal(self, url):
+            logging.debug("Emitting on-url-clicked signal: %s", url)
+
+        def on_before_browse(self, browser, frame, request, **_):
+            url = request.GetUrl()
+            if url == self.NOTEBOOK_URL + "/":
```

Comment:
I have tested few different patterns for `NOTEBOOK_URL` (with different number of slashes at the end) and `GetUrl` allways added one more. It could be related to [this](https://bugs.chromium.org/p/chromium/issues/detail?id=339054) chromium issue but I am not 100% sure.

Reply:
Thanks for looking into this!